src/main/java/com/conveyal/gtfs/api/graphql/TripDataFetcher.java

```diff
@@ -42,10 +54,46 @@ public static Long fromRouteCount(DataFetchingEnvironment dataFetchingEnvironmen
     public static List<WrappedGTFSEntity<Trip>> fromPattern (DataFetchingEnvironment env) {
         WrappedGTFSEntity<Pattern> pattern = (WrappedGTFSEntity<Pattern>) env.getSource();
 
+        Long beginTime = env.getArgument("begin_time");
```

Comment:
Perhaps not.  I could see it being useful in other cases, but we don't need it for NYSDOT atm.